### PR TITLE
add github workflow, pipeline, pipelinerun and tasks

### PR DESCRIPTION
**Pull request by kshapka-bcgov**

### CODE REVIEW

**Comment by mhaswell-bcgov** [comment]
The build config is still using the original NotifyBC repo right? Have we tried building using this repo?
I don't have a good understanding of the OpenShift build process. If we merge this, what will change?

**Comment by kshapka-bcgov** [comment]
> The build config is still using the original NotifyBC repo right? Have we tried building using this repo? I don't have a good understanding of the OpenShift build process. If we merge this, what will change?

The build config will have to be updated to point to the right image.
Once this is merged, the build config [notify-bc ](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/c345c5-tools/buildconfigs/notify-bc)will be triggered when changes to the src folder are merged into v5.

File: helm/deployments/openshift/tekton/build-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      type: string
+  tasks:
+    - name: start-build
+      taskRef:
+        kind: Task
+        name: start-build
+    - name: send-to-rocketchat
+      params:
+        - name: url-secret
+          value: $(params.rocketchat-url-secret)
```
Comment (ShawnTurple) [inline helm/deployments/openshift/tekton/build-pipeline.yaml:18]: is this a helm value? and if so i suspect there is a follow up pr for the tenant repo!

**Review by ShawnTurple — APPROVED** [review]

File: helm/deployments/openshift/tekton/build-pipeline.yaml (hunk 1 above)
Comment (kshapka-bcgov) [reply helm/deployments/openshift/tekton/build-pipeline.yaml:18]: No, it is the name of the openshift secret that the curl task (see that file below) will look up and use. It is provided by the pipeline-run (in this case "rocketchat-url").

**Pull request merged**